1.11.9 breaks in Webpack

We noticed our builds breaking this morning with the error

```Module not found: Error: Cannot resolve module 'instantsearch.js'```

Upon noticing there was a new release we found our package.json was set to install major releases and had installed 1.11.9. 

Reverting to 1.8.13 fixed the error.

I'm somewhat a novice in Webpack so let me know if you need any other details.

--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
   "repository": "algolia/instantsearch.js",
   "files": [
     "dist",
-    "dist-es5-modules"
+    "dist-es5-module"
   ],
   "devDependencies": {
     "autoprefixer": "^6.7.6",

